susemanager-utils/supportutils-plugin-susemanager/susemanager

```diff
@@ -78,4 +78,5 @@ plugin_command "echo \"select count(*) from susesaltevent;\" | spacewalk-sql --s
 plugin_command "/sbin/supportconfig-sumalog $LOG"
 plugin_command "cp /var/log/zypper.log $LOG"
 plugin_command "grep \"Client bootstrap script\" /srv/www/htdocs/pub/bootstrap/*.sh"
+plugin_command "cat /etc/passwd"
```

Comment:
I think people will not like it. passwords should be in shadow, but paranoid people will see this as a problem.
Especially as we print all entries.

Reply:
if we had something like this?
```
    POSTGRES_USER_ENABLED=$(su - postgres -c /bin/true)
    if [ $POSTGRES_USER_ENABLED != "0" ] ; then
        echo "ERROR: Postgres user disabled" 
    else
        echo "Postgres user enabled" 
    fi
``` 
Would it be better?

Reply:
I just found out that spacewalk-debug get this allready.
Check spacewalk-debug/conf/passwd

It only copy the entries for special users not the whole file.
So maybe you can close this. We have this already